CoffeeChat/Controller/InterestsViewController.swift

```diff
@@ -0,0 +1,188 @@
+//
+//  InterestsViewController.swift
+//  CoffeeChat
+//
+//  Created by Phillip OReggio on 2/3/20.
+//  Copyright © 2020 cuappdev. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class InterestsViewController: UIViewController {
+
+    // MARK: - Private View Vars
+    private let titleLabel = UILabel()
+    private let tableView = UITableView(frame: .zero, style: .plain)
+    private let nextButton = UIButton()
+    private let backButton = UIButton()
+
+    // MARK: - Gradients
+    // Fade out affects on the top and bottom of the tableView
+    private let topFade = UIView()
+    private let bottomFade = UIView()
+
+    // MARK: - Data
+    private var interestData: [Interest] = []
+    private var selectedInterests: [Interest] = []
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .backgroundWhite
+
+        titleLabel.text = "What are your interests?"
+        titleLabel.font = .systemFont(ofSize: 24, weight: .bold)
+        view.addSubview(titleLabel)
+
+        tableView.delegate = self
+        tableView.dataSource = self
+        tableView.register(InterestsTableViewCell.self, forCellReuseIdentifier: InterestsTableViewCell.reuseIdentifier)
+        tableView.isScrollEnabled = true
+        tableView.clipsToBounds = true
+        tableView.allowsMultipleSelection = true
+        tableView.showsHorizontalScrollIndicator = false
+        tableView.showsVerticalScrollIndicator = false
+        tableView.separatorStyle = .none
+        let insets = UIEdgeInsets(top: 0, left: 0, bottom: 30, right: 0)
+        tableView.contentInset = insets
+        view.addSubview(tableView)
+        view.addSubview(topFade)
+        view.addSubview(bottomFade)
+
+        nextButton.setTitle("Almost there!", for: .normal)
+        nextButton.layer.cornerRadius = 27
+        nextButton.backgroundColor = .backgroundLightGray
+        nextButton.setTitleColor(.textBlack, for: .normal)
+        nextButton.titleLabel?.font = .systemFont(ofSize: 20)
+        view.addSubview(nextButton)
+
+        backButton.titleLabel?.font = .systemFont(ofSize: 16)
+        backButton.setTitle("Go back", for: .normal)
+        backButton.setTitleColor(.textLightGray, for: .normal)
+        backButton.backgroundColor = .none
+        view.addSubview(backButton)
+
+        setupConstraints()
+    }
+
+    override func viewDidAppear(_ animated: Bool) {
+        let clearColor = UIColor(displayP3Red: 1, green: 1, blue: 1, alpha: 0)
```

Comment:
I think there's `UIColor.clear`?

Reply:
`UIColor.clear` is `UIColor(r: 0, g: 0, b: 0, a: 0)` which makes the fade go towards transparent black instead of transparent white